packages/components/src/autocomplete/style.native.scss

```diff
@@ -0,0 +1,73 @@
+.wrapper {
+	width: 100%;
+	height: $mobile-header-toolbar-height;
+}
+
+.background {
+	height: $mobile-header-toolbar-height;
+	border-top-color: $gray-lighten-30;
+	border-top-width: 1px;
+	background-color: $app-background;
+}
+
+.backgroundDark {
+	border-top-color: $dark-quaternary;
+	border-bottom-width: 1px;
+	border-bottom-color: $dark-quaternary;
+	background-color: $app-background-dark-alt;
+}
+
+.backgroundBlur {
+	width: 100%;
+	height: $mobile-header-toolbar-height;
+}
+
+.content {
+	flex-grow: 1;
+	padding-left: 6px;
+}
+
+.item {
+	flex-direction: row;
+	align-items: center;
+	padding: 5px 12px 5px 8px;
+	margin-right: 2px;
+	margin-top: 3px;
+	margin-bottom: 3px;
+}
+
+.icon {
+	margin-right: 4px;
+	color: $toolbar-button;
+}
+
+.iconDark {
+	color: $gray-20;
+}
+
+.text {
+	color: $toolbar-button;
+}
+
+.textDark {
+	color: $gray-20;
+}
+
+.activeText {
+	text-decoration: underline;
+	text-decoration-color: $toolbar-button;
+}
+
+.activeTextDark {
+	text-decoration: underline;
```

Comment:
This line seems redundant. Since it is already part of the activeText.

Reply:
Nice catch! Fixed with https://github.com/WordPress/gutenberg/pull/29772/commits/4d184ac9e6ab1f4068e1dfa417ce6f8a20c44f0b